adapters/appnexus/appnexus.go

```diff
@@ -375,6 +419,8 @@ func (a *AppNexusAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *ada
 	headers.Add("Content-Type", "application/json;charset=utf-8")
 	headers.Add("Accept", "application/json")
 
+	requestHelper(request, requestExtension, errs)
```

Comment:
The term "helper" is too generic. Please consider renaming to something along the lines of `marshalAndSetRequestExt`.

Reply:
Renamed